tests/functional/codegen/features/test_transient.py

```diff
@@ -570,3 +570,15 @@ def foo() -> (uint256[3], uint256, uint256, uint256):
 
     c = get_contract(main, input_bundle=input_bundle)
     assert c.foo() == ([1, 2, 3], 1, 2, 42)
+
+
+def test_transient_is_state(make_input_bundle):
```

Comment:
let's please add a comment here that we're testing the initialization and how it relates to state